content/en/contribute/code/workflow.md

```diff
@@ -187,6 +187,80 @@ Issues in this column are in the process of being acceptance tested by a Quality
    1. How was the issue verified? Console logs, page source, errors anywhere, execution steps.  LGTM should begin to be rarely used.
    1. Move the issue to "Ready to merge" the notify the original developer to merge the PR.
 
+Once the AT process is done, use the following templates to provide feedback in the ticket.
```

Comment:
My suggestion is to add this as step number 4. in the list above. The existing step 4 would become step 5.

Reply:
That was my plan at the beginning, I didn't add it as step 4 because there is one template for the failed scenario (that is the current step 3) and one for the passed scenario (that is the current step 4), so I was not sure if that might be confusing

Reply:
OK, I understand. Let's keep it this way then, unless someone else has another suggestion.